### DISCUSSION TITLE
계단 작용 문의 드립니다.

### DISCUSSION THREAD

**Discussion by limitless** [naverz/zepeto-studio-kor#633] [5. Q&A]
안녕하세요

빌드잇으로 제작한 맵을 내려 받아 작업중인데 계단으로 인식이 안되고 캐릭터가 그냥 통과를 합니다.
사다리도 빌드잇에서는 잘 되지만  유니티에서는 작용이 안됩니다.

어떻게 해야 될까요?

**Comment by Joyounwoo** [comment]
제페토 월드 제작 시, 기본적으로 제공하는 기능 외의 모든 오브젝트와의 상호작용, 기능, 동작은 직접 구현하셔야 합니다.

계단 통과의 경우 해당하는 계단에 물리 충돌체를 추가해주시면 됩니다.
계단 선택 -> Inspector 창 -> Add Component -> 'Mesh Collider' 입력 & 추가 -> 테스트

사다리의 경우 빌드잇에서 제공하는 기능으로, 제페토 월드에서는 직접 구현하셔야 합니다.
이를 구현하려면 직접 애니메이션 및 로직을 구현하여야 합니다.
구글에 '유니티 3d 캐릭터 애니메이션 만들기' 을 검색하여 자료를 확인해보시면 되겠습니다.

개발 화이팅입니다 :)
